### PR TITLE
Updates sync interval and get cluster information and version.

### PR DESCRIPTION
**Pull request by gabrielbussolo**
# Summary of this proposal

close calyptia/core-operator#111, calyptia/core-operator#113, calyptia/core-operator#110
## Issue(s) number(s)

*Disclaimer: Please do not create a Pull Request without creating an issue first.*

## Checklist for submitter

- [ ] My PR has a related issue/bug number.
- [ ] My PR provides tests
  - [ ] Integration tests are added/passes
  - [ ] Unit tests are added/passes
  - [ ] End-to-end tests added/passes
  - [ ] Static code check added/passes
  - [ ] Linting on documentation added/passes
  - [ ] Does not affect code coverage stats
- [ ] My PR requires updating dependencies
- [ ] My PR has the documentation changes required.

## Checklist for reviewer

- [ ] The proposal fixes a bug/issue or implements a new feature that is well described.
- [ ] The proposal has sufficient test cases that covers the changes.
  - [ ] If changes an API, it does not break backwards compatibility (-1 major version).
- [ ] If integration is required, the proposal has integration tests.
- [ ] The proposal does not break coverage stats
- [ ] The proposal has the required documentation changes.

## Backport

- [ ] Backport to the latest stable release.

### CODE REVIEW

File: cmd/coreinstance/create_core_instance_operator.go [hunk 1]
```diff
@@ -212,6 +212,20 @@ func newCmdCreateCoreInstanceOperator(config *cfg.Config, testClientSet kubernet
 				return err
 			}
 
+			metadata, err := getCoreInstanceMetadata(k8sClient)
+			if err != nil {
+				return err
+			}
+
+			updateCoreInstance := cloud.UpdateCoreInstance{
```
Comment (niedbalski) [inline cmd/coreinstance/create_core_instance_operator.go:220]: Version and metadata are fields accepted by createCoreInstance, why are we using an update?

**Review by niedbalski — CHANGES_REQUESTED** [review]

File: cmd/coreinstance/create_core_instance_operator.go (hunk 1 above)
Comment (gabrielbussolo) [reply cmd/coreinstance/create_core_instance_operator.go:225]: k8s client needs to be instantiated beforehand and one of the dependencies of the client is the core instance already created before. thats why is later. I can change it if needed.

Comment (niedbalski) [reply cmd/coreinstance/create_core_instance_operator.go:225]: @gabrielbussolo I mean, you are already invoking CreateCoreInstance, the CreateCoreInstanceParams also have a metadata and version fields. Why using a subsequent update?

Comment (gabrielbussolo) [reply cmd/coreinstance/create_core_instance_operator.go]: just how about the code was organized, we were using some stuffs for labels on the k8s client after creating the core instance, i changed it

**Review by niedbalski — DISMISSED** [review]
LGTM

**Review by jasmingacic — DISMISSED** [review]
ship it

**Pull request merged**